cli/azd/test/functional/cli_test.go

```diff
@@ -66,11 +66,9 @@ func Test_CLI_Version_PrintsVersion(t *testing.T) {
 
 	rn := os.Getenv("GITHUB_RUN_NUMBER")
 	if rn != "" {
-		runningTestPath := azdcli.GetAzdLocation()
-		cliPath := strings.Split(runningTestPath, "cli")[0]
-		version, err := os.ReadFile(path.Join(cliPath, "cli", "version.txt"))
-		require.NoError(t, err)
-		require.Contains(t, out, strings.ReplaceAll(string(version), "\n", ""))
+		version := os.Getenv("CLI_VERSION")
```

Comment:
What cli is this referring to?  Should we prefix it with something?

Reply:
I would just add some comments here about how this env var is set. 
Something like `CLI_VERSION is set during pipeline step "Set CLI_VERSION". It calls script "Set-CliVersionVariable.ps1" to read the content from "cli/version.txt" to the env var`

A note like this would help folks in the future to understand what's happening here

Reply:
I would also do +1 to Jon's comment.  
We have az cli, github cli, azd cli, etc etc =/   It might be easier here to be a more explicit and use `AZD_CLI_VERSION`

Reply:
I'll keep it as-is for now with renaming as a separate change to unblock PR validation failures. Something like `BUILD_AZD_CLI_VERSION` is probably the most descriptive name I could come up with.